angular/src/lib/date-picker/date-picker-month.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, OnInit } from '@angular/core';
+import { DatePickerService } from './date-picker.service';
+
+@Component({
+  selector: 'cui-date-picker-month',
+  template: `
+    <cui-date-picker-week *ngFor='let _day of weeks' [day]='_day' [month]='month' ></cui-date-picker-week>
+  `,
+  styles: [
+    '.cui-datepicker__week{ display: block !important; }',
+    ':host{display: block !important;}'
+  ],
+  host:{
+    class:'cui-datepicker__month'
+  }
+})
+export class DatePickerMonthComponent implements OnInit {
```

Comment:
Should we do 'implements OnInit; when onInit isn't used?

Reply:
Let me check. Thanks for review